capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
```

Comment:
- [ ] Please change the above code with this to work capitalize pattern to work as expected. `@nameable.correct_name.capitalize`